### PR TITLE
Add Welcome Page

### PR DESCRIPTION
**Pull request by formulahendry**
welcome

image

### CODE REVIEW

File: src/Model/DeviceItem.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import { Command, QuickPickItem, TreeItem } from "vscode";
 
 export class DeviceItem extends TreeItem implements QuickPickItem {
+    public readonly label: string;
```
Comment (adashen) [inline src/Model/DeviceItem.ts]: defined not used?

Comment (formulahendry) [reply src/Model/DeviceItem.ts]: DeviceItem needs to implement label property in QuickPickItem

File: src/azureIoTExplorer.ts [hunk 2]
```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT license.
 
 "use strict";
+import * as path from "path";
```
Comment (adashen) [inline src/azureIoTExplorer.ts]: import used?

File: src/extension.ts [hunk 3]
```diff
@@ -10,6 +10,7 @@ import { Executor } from "./executor";
 import { DeviceTwinCodeLensProvider } from "./providers/deviceTwinCodeLensProvider";
 import { ModuleTwinCodeLensProvider } from "./providers/moduleTwinCodeLensProvider";
 import { TelemetryClient } from "./telemetryClient";
+import { Utility } from "./utility";
```
Comment (adashen) [inline src/extension.ts]: Utility referenced?

**Pull request merged**